File: examples/src/Slides/Views/ManagePresenetationNormalViewState.cs
import aspose.slides as slides
using Aspose.slides.Examples.CSharp
using Aspose.slides.Export
using System
using System.Collections.Generic
using System.Linq
using System.text
using System.Threading.Tasks

namespace CSharp.slides.Views
{
    class ManagePresenetationNormalViewState
    {
        public static void Run() {

            #ExStart:ManagePresenetationNormalViewState
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Slides_Views()

            with slides.Presentation() as pres:
            {
                pres.ViewProperties.NormalViewProperties.HorizontalBarState = SplitterBarStateType.Restored
                pres.ViewProperties.NormalViewProperties.VerticalBarState = SplitterBarStateType.Maximized

                pres.ViewProperties.NormalViewProperties.RestoredTop.AutoAdjust = True
                pres.ViewProperties.NormalViewProperties.RestoredTop.DimensionSize = 80
                pres.ViewProperties.NormalViewProperties.ShowOutlineIcons = True

                pres.save(dataDir+ "presentation_normal_view_state.pptx", slides.export.SaveFormat.PPTX)
            }

            #ExEnd:ManagePresenetationNormalViewState
        }
    }
}

File: examples/src/Slides/Views/ManagePresenetationNormalViewState.py
import aspose.slides as slides


#ExStart:ManagePresenetationNormalViewState
# The path to the documents directory.
dataDir = "./examples/data/"
outDir = "./examples/out/"

with slides.Presentation() as pres:
    pres.view_properties.normal_view_properties.horizontal_bar_state = slides.SplitterBarStateType.RESTORED
    pres.view_properties.normal_view_properties.vertical_bar_state = slides.SplitterBarStateType.MAXIMIZED

    pres.view_properties.normal_view_properties.restored_top.auto_adjust = True
    pres.view_properties.normal_view_properties.restored_top.dimension_size = 80
    pres.view_properties.normal_view_properties.show_outline_icons = True

    pres.save(outDir+ "presentation_normal_view_state.pptx", slides.export.SaveFormat.PPTX)

#ExEnd:ManagePresenetationNormalViewState
